elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryThreadTest.java

```diff
@@ -61,7 +61,10 @@ public void testProcessQueryJsonApi() {
         when(queryObj.getQueryType()).thenReturn(QueryType.JSONAPI_V1_0);
         when(elide.get(anyString(), any(), any(), anyString())).thenReturn(response);
         when(response.getResponseCode()).thenReturn(200);
-        when(response.getBody()).thenReturn("ResponseBody");
+
+        String testStr = "{\"data\": []}";
+        String testStr1 = "responseBody";
```

Comment:
testStr1 is not used in the test case anywhere.

Reply:
Removed the unused String

Reply:
Also, Added a test to check the recordCount in the AsyncIT integration tests